contrib/internal/httputil/trace.go

```diff
@@ -26,6 +26,7 @@ func TraceAndServe(h http.Handler, w http.ResponseWriter, r *http.Request, servi
 		tracer.ResourceName(resource),
 		tracer.Tag(ext.HTTPMethod, r.Method),
 		tracer.Tag(ext.HTTPURL, r.URL.Path),
+		tracer.Tag("http.host", r.URL.Host),
```

Comment:
We should only set this tag if `r.URL.Host` is not empty, we don't want the tag there otherwise, right? What do you think?

Reply:
i thought about it and felt like it wasnt a big deal to have an empty value, and actually thought it might be useful. However making the change to add only if present